schema/reportdb/uyuni-reportdb-schema.spec

```diff
@@ -0,0 +1,96 @@
+#
+# spec file for package susemanager-reportdb-schema
+#
+# Copyright (c) 2021 SUSE LLC
+# Copyright (c) 2008-2018 Red Hat, Inc.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
+%{!?fedora: %global sbinpath /sbin}%{?fedora: %global sbinpath %{_sbindir}}
+
+Name:           uyuni-reportdb-schema
+Summary:        Report DB SQL schema for Spacewalk server
+License:        GPL-2.0-only
+Group:          Applications/Internet
+
+Version:        4.3.4
+Release:        1
+Source0:        %{name}-%{version}.tar.gz
+Source1:        %{name}-rpmlintrc
+
+URL:            https://github.com/uyuni-project/uyuni
+BuildArch:      noarch
+BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+
+Requires:       susemanager-schema-core
+Requires:       susemanager-schema-sanity
+
+Provides:       uyuni-reportdb-schema = %{version}
+
+%if 0%{?suse_version}
+BuildRequires:  fdupes
+%endif
+
+%define rhnroot /etc/sysconfig/rhn/
+%define postgres %{rhnroot}/postgres
+
+%description
+susemanager-reportdb-schema is the SQL schema for the SUSE Manager server.
+
+%install
+install -m 0755 -d $RPM_BUILD_ROOT%{rhnroot}
+install -m 0755 -d $RPM_BUILD_ROOT%{postgres}
+#TODO Install SQL Script
+#install -m 0644 postgres/main.sql $RPM_BUILD_ROOT%{postgres}
```

Comment:
This means it will install the `main.sql` into `/etc/sysconfig/rhn/postgres/`. But this file will collide with the other schema package, if installed. We need to change this to `%{rhnroot}/reportdb` or `%{rhnroot}/postgres/reportdb` (not sure which one is better). The same goes for the `schema-upgrade` directory below.